packages/material-ui/src/Autocomplete/Autocomplete.js

```diff
@@ -69,7 +69,7 @@ export const styles = (theme) => ({
         padding: '6px 0',
       },
     },
-    '&[class*="MuiInput-root"][class*="MuiInput-marginDense"]': {
+    '&[class*="MuiInput-root"][class*="MuiInput-sizeSmall"]': {
```

Comment:
This is a good example that we missed listing this in the breaking changes list. Just to me compelte.

Reply:
We can anyway argue that it is obvious from the text in the migration guide, but I'd say it is better if we list the change there.

Reply:
okay, @mnajdova it it just for the breaking change list, so no need to changes, right?

Reply:
Yeah, sorry if we decide to list it, it should be added in the `migration-v4.md`  https://github.com/mui-org/material-ui/pull/23832/files#diff-c66b437e5d952a0998e624e32687b4f3114020624f2fee56d4894f27579874f3R1001

Reply:
@oliviertassinari what do you think?

Reply:
I have tried to cover this part with

> Rename `marginDense` and `inputMarginDense` classes to `sizeSmall` and `inputSizeSmall` to match the prop.

Assuming that it would impact global classes, theme overrides, and the `classes` prop.